api/src/routers/past-meals.js

```diff
@@ -0,0 +1,17 @@
+import express from "express";
+import knex from "../database_client.js";
+
+const pastMealsRouter = express.Router();
+
+
+pastMealsRouter.get('/', async (req, res) => {
+    try {
+        const now = new Date();
+        const past_meals =  await knex('Meal').select('*').where('when', '<', now);
```

Comment:
you think you can make a function for the `last_meal` and `past_meal` functionality, where the input to the function is `<` or `>` as parameter? then both of your endpoints can use this function